src/m365/spo/commands/orgassetslibrary/orgassetslibrary-add.ts

```diff
@@ -38,7 +45,8 @@ class SpoOrgAssetsLibraryAddCommand extends SpoCommand {
     this.telemetry.push((args: CommandArgs) => {
       Object.assign(this.telemetryProperties, {
         cdnType: args.options.cdnType || 'Private',
-        thumbnailUrl: typeof args.options.thumbnailUrl !== 'undefined'
+        thumbnailUrl: typeof args.options.thumbnailUrl !== 'undefined',
+        orgAssetType: typeof args.options.orgAssetType !== 'undefined'
```

Comment:
Since this value is an enum, it would be helpful (and is still safe) to track it and not just look if it's been set.